packages/nodes-base/nodes/Medium/Medium.node.ts

```diff
@@ -0,0 +1,448 @@
+							},
+							{
+								name: 'cc-40-by-nc',
+								value: 'cc-40-by-nc',
+							},
+							{
+								name: '“cc-40-by-nc-nd',
+								value: '“cc-40-by-nc-nd',
+							},
+							{
+								name: 'cc-40-by-nc-sa',
+								value: 'cc-40-by-nc-sa',
+							},
+							{
+								name: 'cc-40-zero',
+								value: 'cc-40-zero',
+							},
+							{
+								name: 'public-domain',
+								value: 'public-domain',
+							},
+
+
+						],
+						description: 'Tags separated by comma to classify the post. Only the first three will be used. Tags longer than 25 characters will be ignored.',
+					},
+				],
+			},
+
+		]
+
+	};
+	methods = {
+		loadOptions: {
+			// Get all the available publications to display them to user so that he can
+			// select them easily
+			async getPublications(this: ILoadOptionsFunctions): Promise<INodePropertyOptions[]> {
+				const returnData: INodePropertyOptions[] = [];
+				//Get the User Id
+				const user = await mediumApiRequest.call(
+					this,
+					'GET',
+					`/me`,
+					{},
+					{}
+				);
+				const userId = user.data.id;
+				//Get all publications of that user
+				const publications = await mediumApiRequest.call(
+					this,
+					'GET',
+					`/users/${userId}/publications`,
+					{},
+					{}
+				);
+				const publications_list = publications.data;
+				for (const publication of publications_list) {
+					const publicationName = publication.name;
+					const publicationId = publication.id;
+					returnData.push({
+						name: publicationName,
+						value: publicationId,
+					});
+				}
+				return returnData;
+			},
+
+		},
+	};
+
+	async execute(this: IExecuteFunctions): Promise<INodeExecutionData[][]> {
+
+		const items = this.getInputData();
+		const returnData: IDataObject[] = [];
+
+		let operation: string;
+		let resource: string;
+
+		// For POST
+		let bodyRequest: IDataObject;
+		// For Query string
+		let qs: IDataObject;
+		let responseData;
+
+		for (let i = 0; i < items.length; i++) {
+			qs = {};
+
+			resource = this.getNodeParameter('resource', i) as string;
+			operation = this.getNodeParameter('operation', i) as string;
+
+			if (resource === 'post') {
+				//https://github.com/Medium/medium-api-docs
+				if (operation === 'create') {
+					// ----------------------------------
+					//         post:create
+					// ----------------------------------
+
+					const title = this.getNodeParameter('title', i) as string;
+					const contentFormat = this.getNodeParameter('contentFormat', i) as string;
+					const content = this.getNodeParameter('content', i) as string;
+					bodyRequest = {
+						tags: [],
+						title,
+						contentFormat,
+						content,
+
+					};
+					const additionalFields = this.getNodeParameter(
+						'additionalFields',
+						i
+					) as IDataObject;
+					if (additionalFields.tags) {
+						const tags = additionalFields.tags as string;
+						bodyRequest.tags = tags.split(',').map(item => {
+							return parseInt(item, 10);
+						});
```

Comment:
No need to expand the body:
```ts
tags.split(',').map(item => parseInt(item, 10);
```

Reply:
Also, too tired to verify right now, but please double check if you really need the radix arg here.

Reply:
Radix arg was added to one of the nodes before.